rabies/analysis_pkg/analysis_functions.py

```
@@ -185,48 +185,44 @@
-    optimize_NPR_dict = traits.Dict(
-        desc="Dictionary with options for optimizing NPR convergence.")
+    CPCA_dict = traits.Dict(
+        desc="Dictionary with options for optimizing CPCA convergence.")
     network_weighting = traits.Str(
         desc="Whether to derive absolute or relative (variance-normalized) network maps.")
     figure_format = traits.Str(
         desc="Select file format for figures.")
 
-class NeuralPriorRecoveryOutputSpec(TraitedSpec):
-    NPR_prior_timecourse_csv = File(
+class ComplementaryPCAOutputSpec(TraitedSpec):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)